_posts/en/newsletters/2018-07-10-newsletter.md

```diff
@@ -7,6 +7,11 @@ layout: page
 lang: en
 version: 1
 ---
+This week's newsletter includes news and action items about minimum fees and
+the upcoming Bitcoin Core release, a special feature on a Schnorr signature
+proposal, and a write-up of the recent Building on Bitcoin conference in
+Lisbon. *Divertam-se!*
```

Comment:
I had to Google "Divertam-se", which is not necessarily a bad thing, but it feels not quite right to me.  Up to you if you want to keep it.

Reply:
You don't speak Portuguese? ;)

Will remove.